### PR TITLE
add flags to global ocean tests

### PR DESCRIPTION
**Pull request by mark-petersen**
Turn on all terms in global ocean tests, to exercise as much code as possible on QU240 restart, partition, thread tests, etc.

### CODE REVIEW

File: compass/ocean/tests/global_ocean/namelist.forward [hunk 1]
```diff
@@ -3,6 +3,9 @@ config_block_decomp_file_prefix = 'graph.info.part.'
 config_time_integrator = 'split_explicit'
 config_use_mom_del2 = .true.
 config_use_mom_del4 = .true.
+config_use_tracer_del2 = .true.
+config_use_tracer_del4 = .true.
```
[the remark this answers is not in the document]
Comment (xylar) [reply compass/ocean/tests/global_ocean/namelist.forward:7]: Here are some typical values used in POP for reference:
https://github.com/ESCOMP/POP2-CESM/blob/9f2a48328dda6bd98be74f4ccfc13248b1b13868/bld/namelist_files/namelist_defaults_pop.xml#L1875-L1880
I'm not even sure if we have the same sign convention.

**Comment by xylar** [comment]
My suggestion above will also modify QUwISC240 test cases.  I think that's okay.  We would need to make code modifications if we want different options for QU240 vs QUwISC240.

**Comment by mark-petersen** [comment]
Revised based on comments above, tested with `pr` test suite with grizzly debug gnu, passes. This will break bfb comparisons of QU240 after merge into compass.

I'd like to take this opportunity and look at all the other features we can turn on for QU240. Basically "the works", anything that does not require a forcing file or conflict with an error with other settings. It doesn't have to even make sense - like cvmix and gotm both running, unless they overwrite each other's fields.

I'm considering:
```
    config_Rayleigh_friction = .false.
    config_Rayleigh_damping_depth_variable = .false.
    config_Rayleigh_bottom_friction = .false.
    config_use_gotm = .false.
    config_use_variable_drag = .false.
    config_use_tidal_forcing = .false.
    config_use_tidal_potential_forcing = .false.
    config_use_vegetation_drag = .false.
    config_use_vegetation_manning_equation = .false.
    config_use_frazil_ice_formation = .false.
    config_use_implicit_bottom_roughness = .false.
    config_use_implicit_bottom_drag_variable = .false.
    config_use_implicit_bottom_drag_variable_mannings = .false.
    config_use_explicit_bottom_drag = .false.
    config_use_topographic_wave_drag = .false.
    config_use_wetting_drying = .false.
```

@xylar do you prefer if these changes are in this PR or separate ones?

**Review by xylar — COMMENTED** [review]
This looks great!  I just want to run a few tests.

**Comment by mark-petersen** [comment]
Thanks, will do. I'll work on this further next week.

**Comment by mark-petersen** [comment]
@xylar this is ready for you to test.

**Pull request merged**